### PR TITLE
ACC-539: Visible labels 🐿 v2.13.0

### PR DESCRIPTION
**Pull request by rosario**
Ticket: https://financialtimes.atlassian.net/browse/ACC-539

### CODE REVIEW

File: package.json [hunk 1]
```diff
@@ -38,5 +35,12 @@
   },
   "dependencies": {
     "handlebars": "^4.1.2"
+  },
+  "husky": {
+    "hooks": {
+      "commit-msg": "node_modules/.bin/secret-squirrel-commitmsg",
+      "pre-commit": "node_modules/.bin/secret-squirrel",
+      "pre-push": "make verify -j3"
+    }
```
Comment (rosario) [inline package.json:44]: It was done automatically by `make build`

**Comment by rosario** [comment]
Screen Shot 2020-07-24 at 14 49 21

File: main.scss [hunk 2]
```diff
@@ -205,19 +203,19 @@
 		// Force the input and label into a vertical orientation
```
Comment (nickcolley) [inline main.scss:203]: Having a look at this I believe we can remove a lot of code to achieve the same thing as a lot of this code only exists to adjust the default stacked appearance.

File: src/survey-builder.js [hunk 3]
```diff
@@ -42,20 +42,15 @@ function buildMultipleChoiceQuestion (question) {
...
-		let textVisibility = '';
-		// if not the first or the last element
-		if ( !(choiceId === choices[0][0] || choiceId === choices[choices.length-1][0]) ) {
-			textVisibility = 'hidden-label';
-		}
+	choices.forEach( ([choiceId, choice]) => {
 		const fieldId = `choice-${choiceId}-${~~(Math.random()*0xffff)}`;
 
 		html.push(
 			`<div class="n-feedback__question-radio__choice-container">
```
Comment (nickcolley) [inline src/survey-builder.js:49]: When reviewing the origami component I believe we can simplify the markup here: https://www.ft.com/__origami/service/build/v2/demos/o-forms@8.3.10/radio-round?brand=master

**Comment by nickcolley** [comment]
Just an update I spoke to #design-ops and they're happy for this to go ahead. Happy to pair with you or chat through any of this feedback over Slack or on a call, let me know.

**Comment by nickcolley** [comment]
Going to do some browser / screen reader testing.

**Comment by nickcolley** [comment]
Screenshot 2020-07-28 at 14 35 50

Screenshot 2020-07-28 at 14 36 04

File: src/survey-builder.js [hunk 4]
```diff
@@ -183,11 +175,11 @@ function buildSurvey (surveyData, surveyId, domain) {
 
 			${feedbackQuestion ? buildFooter(feedbackQuestion) : ''}
 
-			<div class="n-feedback__center-block"><p class="n-feedback__survey__button-bar">
```
Comment (nickcolley) [inline src/survey-builder.js:186]: I think we might want to bring this center-block container back in the places it was removed. It was only `n-feedback__question-radio__container` that was unnecessary.

**Comment by rosario** [comment]
Screen Shot 2020-07-28 at 15 26 33

Test on `next-profile`. All good.

**Pull request merged**